docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -62,62 +56,234 @@ in 2012. This is the latest National Guideline for the Management of Acute Malnu
 
+.. _1.1:
+
 Health system delivery
 ++++++++++++++++++++++
 
+Interventions for wasting treatment are delivered through different levels of the health system.
+
+:underline:`Community and household level`
+ 
+ Community outreach ensures early identification of SAM and MAM cases. Community outreach also aims to empower communities and families to understand the causes of malnutrition, and prevent and manage acute malnutrition in their communities. Health Extension Workers (HEWs) collaborate with the community-based structures such as health committees and engage with the Health Development Army (HDA)/Health Development Group (HDG) to screen and refer cases to the appropriate service for treatment.
+
+:underline:`Health post level`
+
+ The HPs provide primary health care services such as disease prevention and control, hygiene and environmental sanitation, family health services, and health education and communication. The service provider at the HP works closely with the network of the HDA/HDG to facilitate the management of SAM and MAM. The
+ service provider diagnoses acute malnutrition and provides Outpatient Therapeutic Programme (OTP) and Targeted Supplementary Feeding Programme (TSFP) services. They also determine patients who have medical complications and refer them to the Stablisation Centre (SC).
+
+:underline:`Health center level`
+
+ The HC provides 24-hour SC for the management of SAM with medical complications. The HC also manages and provides technical support to a cluster of one to five HPs within its vicinity.
 
-0-6 months
-++++++++++
+:underline:`Hospital level`
 
-Infants 0-6 months: Recent weight loss or failure to gain weight OR Ineffective feeding (attachment, position, and suckling)
+ Woreda, Zonal, Regional and referral Hospitals provide higher level referral points where further care can be provided to patients with SAM and more complex medical complications. The referral Hospitals have the facilities and expertise to manage situations that may require administration of oxygen, blood transfusion, and other critical care.
 
+.. _2.0:
 
-Severe acute malnutrition
-~~~~~~~~~~~~~~~~~~~~~~~~~
+Assessing and classifying acute malnutrition
+--------------------------------------------
 
-With medical complications
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-Without medical complications
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The classification of SAM and MAM cases is derived from the 2019 National Guideline for the Management of Acute Malnutrition. [EMOH]_
 
+:download:`2019 guidelines<guidelines_2019.pdf>`
 
-6-60 months
-+++++++++++
+.. note:: 
+  
+  - Based on the guideline, we will need to make some modelling decisions on how we classifiy SAM, MAM, NONE and think about how it may bias our results. 
+
+  - The ORs and ANDs in the classifications are not very clear or consistent in the guideline. It would be useful if we can clarify with a clinician when it should be OR/AND? 
```

Comment:
Let's ask Rafael Lozano

Reply:
Excellent! I also know a pediatrician at the center for global child health in Amsterdam specialized in malnutrition, working mostly in Malawi should we need.